src/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -6,7 +6,10 @@
 
   <PropertyGroup>
     <Version>$(SharedFrameworkNugetVersion)</Version>
```

Comment:
While not exactly related should this also be ProductVersion instead of SharedFrameworkNugetVersion?